libs/libtasn1/Makefile

```diff
@@ -35,12 +36,13 @@ define Package/libtasn1/description
  Distinguish Encoding Rules (DER) manipulation.
 endef
 
-TARGET_CFLAGS += $(FPIC)
```

Comment:
why was the $(FPIC)  removed?

Reply:
It gets passed by default. Verified with checked.sh